locust/shape.py

```diff
@@ -7,7 +7,8 @@ class LoadTestShape(object):
     during a load test.
     """
 
-    start_time = time.monotonic()
+    def __init__(self):
+        self.start_time = time.monotonic()
```

Comment:
I don't think this is ever called? Because the class is never instantiated, it's just declared [eg](https://github.com/locustio/locust/blob/master/examples/custom_shape/step_load.py#L17), so `__init__ ` is never run.

Reply:
Isn't it instantiated here: https://github.com/locustio/locust/blob/08d5173c2551e2f8c16404345816482f5af25394/locust/main.py#L93 (making the `shape_class` variable name a bit of a misnomer...)

Reply:
Ah yes, you are right. I wrote that, I should know!